Reorganize docs

amberarrow opened the pull request:
**For Review Only** @tweise Please take a look; the Presentations section has just a couple of entries right now -- I'll add more if we are OK with this style/format. Couple of notes:
1. Added a short synopsis for each item so the reader has an idea of what a link has.
2. Removed names of people and dates for uniformity.

Commit:
Reorganize docs

tweise commented on src/md/docs.md:
```diff
@@ -1,50 +1,65 @@
+
 - [Example Applications](https://github.com/DataTorrent/examples)
+A `git` repository with sample code for specialized tutorials covering a
+a variety of topcs such as a high-performance key-value store (HDHT), custom
```
topics (typo)

tweise commented:
Looks like a few things fell off the cliff here. We should list the intro webinar under presentations. We should also link the slides from the fault-tolerance webinar:  http://www.slideshare.net/ApacheApexOrganizer/webinar-fault-toleranceandprocessingsemantics

Should list trouble shooting guide.

Should also list buffer server presentations, but I don't have the slideshare link.

tweise commented:
The versioned documents for core and malhar could be a bit more prominent?

tweise commented:
Another good candidate:  http://www.slideshare.net/DevendraVyavahare/windowing-in-apex

amberarrow commented:
They didn't even approach the cliff :-)

As I mentioned in my initial comment above, this was just to confirm that we're OK with this reorg style.
I'll be adding many more items to the "Presentations" section, drawn from the spreadsheet and elsewhere.

tweise commented:
You are right, I checked at the beach, nothing there :-)

General format is good, except the versioned docs at the beginning, as already commented.

tweise commented:
@sashadt any comments?

sashadt commented:
Format looks good!  It's more descriptive, and will help with SEO.

Let me know if you were planning any other changes, and we can merge.

amberarrow commented:
Don't merge yet, more content coming.

tweise commented:
links to datatorrent.com should be ref nofollow

Commit:
Added more links and descriptions

Commit:
Merge branch 'master' into reorg-docs

amberarrow commented:
Addressed comments and added rest of the content. Couldn't find any presentations dedicated to the buffer server.
@sashadt This is now ready for review and merge.

tweise commented:
This may be a good link to place on this page or elsewhere on the site for general ASF background:

https://blogs.apache.org/foundation/entry/celebrating_17_years_of_the

Commit:
Added link to ASF blog

tweise commented:
@amberarrow There are a few unexpected line breaks in the final page. Please check.
